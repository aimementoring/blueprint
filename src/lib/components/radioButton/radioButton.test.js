import React from 'react';
import renderer from 'react-test-renderer';
import RadioButtons from './radioButton';

describe('PhoneInput', () => {
  it('renders properly', () => {
    const radioButtonsOptions = [
      {
        value: 'notReturningMentor',
        text: 'Yes this will be my first year as an AIME mentor and I can’t flippin’ wait!',
      },
      {
        value: 'returningMentor',
        text: 'I’ve mentored with AIME before and will register with the same email I used then. I also can’t flippin’ wait!',
      },
    ];
    const tree = renderer
      .create(
        <RadioButtons
          className=""
          onChangeFunction={() => { }}
          inputName="returningMentor"
          options={radioButtonsOptions}
          value="returningMentor"
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
