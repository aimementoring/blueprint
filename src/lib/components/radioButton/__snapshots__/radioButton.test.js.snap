// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhoneInput renders properly 1`] = `
<div
  className="theme-base"
>
  <div
    className=" radioButtonContainer"
  >
    <div>
      <input
        checked={false}
        id="notReturningMentor"
        onChange={[Function]}
        type="radio"
        value="notReturningMentor"
      />
      <label
        htmlFor="notReturningMentor"
      >
        Yes this will be my first year as an AIME mentor and I can’t flippin’ wait!
      </label>
    </div>
    <div>
      <input
        checked={true}
        id="returningMentor"
        onChange={[Function]}
        type="radio"
        value="returningMentor"
      />
      <label
        htmlFor="returningMentor"
      >
        I’ve mentored with AIME before and will register with the same email I used then. I also can’t flippin’ wait!
      </label>
    </div>
  </div>
</div>
`;
